src/components/ScaffoldVuer.vue

```diff
@@ -408,6 +408,11 @@ export default {
     },
     helpMode: function(val){
       this.setHelpMode(val);
+    },
+    sceneData:{
+      handler:  function(scene){
+        this.$emit('current-time', scene.currentTime);
+      }, deep: true
```

Comment:
Using 
```
"sceneData.currentTime": function(val){
...
}
```
will avoid triggering callback when other members of sceneData change.

May I also suggest changing the callback name to "timeChanged" from "current-time"?
Thanks.

Reply:
Thank you and sorry @alan-wu, missed this one! Fixed now.